src/eval-frame/components/panes/console/console-message.jsx

```diff
@@ -15,12 +14,13 @@ const MessageContainer = styled(ConsoleContainer)`
 `;
 
 const MessageBody = styled("div")`
-  font-family: ${THEME.client.console.fontFamily};
```

Comment:
all for removing it if it makes things easier, but I'm noticing that the individual child reps aren't handling their own styling, so I would suggest going through the other console components and adding them. I noticed for python, for instance, that the font is not set to `THEME.client.console.fontFamily`. Should be an easy fix!

![Screen Shot 2019-04-10 at 3 01 50 PM](https://user-images.githubusercontent.com/95735/55918285-64c7ea00-5ba7-11e9-9b8d-1c38f5898750.png)

For reference, here it is on master:

![Screen Shot 2019-04-10 at 3 02 47 PM](https://user-images.githubusercontent.com/95735/55918303-790be700-5ba7-11e9-9b2d-b2ffbb5cfeec.png)

Reply:
ah! thanks for catching that!

Reply:
looking at this again, i don't think that "system messages" need to by monospaced, just things that "look like code" as it were. i actually prefer the system messages to not look like code -- i think it sets them apart visually.

anyhoo, i'm not attached to that viewpoint, especially since we have already decided that enhanced console handling will be coming up soon, which will give us another chance to mark things out visually... but since that effort will be coming up soon anyway, and since i think this look might actually be _better_ i'm going to hold off for now and we can revisit in a couple weeks when we dig into the console again.

Reply:
Don't agree with this assessment, which is why I had it explicitly set to monospace before (and started this comment thread). No other console system we or our users use makes this distinction, and I'm not sure why we're doing that here. But I suppose it can get addressed some other time since this has merged.

Reply:
sure sure -- no other console does that, but i think that's probably because in most consoles either (a) they can _only_ display a single font, which has historically been fixed-width from the olden days of green pixel terminal, or (b) they are trying to _emulate_ the appearance of the terminals from the ancient days. But there is nothing sacrosanct about that appearance, and i suspect that it's mostly just a historical accident that most terminals today follow that tradition. it seems that there may be advantages having code (and code like things) be fixed-width, but for messages it really doesn't matter at.

I don't care about this all that much if you have strong feelings, but there is for sure a semantic distinction between our system/app messages and our code inputs and outputs. how we want to indicate that distinction to our users is up to us; maybe we don't need to do anything at all, and it's best to have system messages blend in with the rest of the console. But the fact that other other console systems have not drawn that distinction is beside the point -- most text-based consoles do not have the ability to make that kind of distinction clear with anything other than text color (which is frequently used to indicate different kinds of content). no other console system we use embeds interactive table viewers either -- we can always try something new when what has come before is the result of historical accident and doing something different might be better

so ya: whatever you want to do when we revisit the console will be fine with me :-) but i think it would be nice to recognize this distinction and to go in with an open mind as we think about whether and how to signify that distinction to our users. we can do what we want: we might decide that fixed-width should always be used in the console because it signifies "you are looking at something terminal-like" (in which case we might want to apply that styling in throughout the table as well?), or we might decide that fixed-width specifically means "you are looking at something code-like" -- inputs and outputs.

an empire of signs! let's keep our semiotics on point :-)